packages/code-infra/src/utils/changelog.mjs

```diff
@@ -39,8 +40,10 @@ export async function findLatestTaggedVersion(opts) {
   const $$ = $({ cwd: opts.cwd });
   const fetchAll = opts.fetchAll ?? true;
   if (fetchAll) {
-    // Fetch all tags from all remotes to ensure we have the latest tags.
-    await $$`git fetch --tags --all`;
+    const { remoteName } = await getRepositoryInfo();
+    // Fetch all tags from the mui remote to ensure we have the latest tags.
+    // --force to update any existing tags that may have changed to avoid the clobering error.
+    await $$`git fetch --tags --force ${remoteName}`;
```

Comment:
is it possible to use the remote url here instead of the name? could be more robust to avoid building on top of local git settings. it would prevent is from having to pass `remoteName` in the first place.

Reply:
Updated the logic of `getRepositoryInfo` to go through all remotes (instead of accepting a parameter) and return the one with `mui` username.